synapse/lib/lmdbslab.py

```diff
@@ -573,7 +573,7 @@ class Slab(s_base.Base):
     synctask = None
     syncevnt = None  # set this event to trigger a sync
 
-    COMMIT_PERIOD = 0.2  # time between commits
+    COMMIT_PERIOD = os.environ.get('SYN_SLAB_COMMIT_PERIOD', 0.2)  # time between commits
```

Comment:
Env vars are strings, does this get normalized or cast somewhere?

Reply:
Fixed.